lsm.pyi

```diff
@@ -17,6 +17,18 @@ SEEK_GE: int
 SEEK_LEFAST: int
 
 
+class _DBStructurePage(TypedDict):
+    first: int
+    last: int
+    root: int
+
+
+class _DBStructure(TypedDict):
```

Comment:
Why not `NamedTuple`? It should be cheaper to use.

Reply:
That's just for typechecks, under the hood this is simple dict